### PR TITLE
Update codeowners

### LINKED ISSUE #65: Enable team review requests for this repo
Currently this repo has relatively rigid review requirements. Merging is blocked until 2 reviews are submitted by specific people listed as CODEOWNERS.

This is especially limiting for any changes in the top-most level of the repo, such as edits to the README file (e.g. #64), because the list of CODEOWNERS is small and somewhat arbitrary.

I suggest it would be better to adopt the GitHub Teams model that we are currently using for https://github.com/ProjectPythia/pythia-foundations

### LINKED ISSUE #79: Enable automation of PR review assignments across repos
Investigate and hopefully enable GitHub's "round robin" reviewer assignment for all repos as described here: 

https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team#routing-algorithms

### PR DESCRIPTION
**Pull request by andersy005**
Fixes #65. Do we want to close #79, too?

### CODE REVIEW

**Comment by clyne** [comment]
Will we have round-robin scheduling enabled after this? Thanks.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,8 +1,2 @@
 # GLOBAL OWNER
-*   @kmpaul
-
-# CI OWNER
-/ci/   @andersy005 @kmpaul
-
-# CONTENT OWNER
-/content/   @jukent @kmpaul @ktyle @clyne @brian-rose
+*   @ProjectPythia/infrastructure @ProjectPythia/education
```
Comment (kmpaul) [inline .github/CODEOWNERS:2]: Should this be used instead?

```suggestion
*   @ProjectPythia/infrastructure

/content/ @ProjectPythia/infrastructure @ProjectPythia/education
```

**Review by kmpaul — CHANGES_REQUESTED** [review]
What do you think about this?

**Review by clyne — APPROVED** [review]

**Review by brian-rose — APPROVED** [review]

**Review by kmpaul — APPROVED** [review]
Thanks, @andersy005!

**Comment by brian-rose** [comment]
Well, this is annoying. The builds are [completing on readthedocs](https://readthedocs.org/projects/projectpythia/builds/14015731/), but forever reported as "Pending" here.

**Comment by clyne** [comment]
That would explain what is going on with #97

**Pull request merged**